src/operator/context/account.tsx

```diff
@@ -87,6 +102,28 @@ export const AccountProvider: FC<Props> = React.memo(({children}) => {
     handleGetAccount()
   }, [handleGetAccount, accountID])
 
+  const handleConvertAccountToContract = useCallback(
+    async contractStartDate => {
+      try {
+        setConvertStatus(RemoteDataState.Loading)
+        const resp = await patchOperatorAccountsConvert({
+          accountId: accountID,
+          data: {contractStartDate},
+        })
```

Comment:
@wdoconnell I think we should consider moving this into an API layer and out of the context layer. What do you think? Should it happen in this PR (which is already doing a lot of work and has been in progress for over a week) or is that something we should follow up on?